tests/tests_unit/test_api/test_files.py

```diff
@@ -179,6 +179,16 @@ def test_list_root_ids_list(self, mock_files_response):
             "filter": {"rootAssetIds": [{"id": 1}, {"id": 2}, {"externalId": "a"}]},
         } == jsgz_load(calls[0].request.body)
 
+    def test_list_with_time_dict(self, mock_files_response):
+        FILES_API.list(created_time={"min": 20})
```

Comment:
if the user can do this, is the type hint in the function incorrect?

Reply:
I would say that the type hint is correct, but that our api is permissive of still using the old code. This test is here to confirm that user can still use the old way (not breaking code), while we encourage going over to the more type safe way.

Reply:
we do allow this in e.g. all filters and it's a pretty standard we support. i think typehints there have Union as well

Reply:
Ah, ok. Can allow union for now.